lib/search/query_components/autocomplete.rb

```diff
@@ -1,30 +1,15 @@
 module QueryComponents
   class Autocomplete < BaseComponent
-    AUTOCOMPLETE_FIELD = "title.edgengram".freeze
+    AUTOCOMPLETE_FIELD = "autocomplete".freeze
 
     def payload
       {
-        "bool" => {
```

Comment:
Suggest query does not make use of the bool functionality in Elasticsearch. Refined method shouldn't require these  exceptions to avoid certain types of document, however we should be aware of the need to enable context-based suggestions in the future.